### ISSUE TITLE
Using SWRAID creates ESP as RAID

### ISSUE THREAD

**Issue by FelixJacobi** [hetzneronline/installimage#106]
Hi,

I used the follow config:

~~~
DRIVE1 /dev/nvme0n1
DRIVE2 /dev/nvme1n1
HOSTNAME some-hostname.example
PART /boot/efi esp 256M
SWRAID 1
SWRAIDLEVEL 1
PART /boot  ext4      1G
PART lvm    vgroot    all
LV vgroot   root   /        ext4         all
IMAGE /root/.oldroot/nfs/install/../images/Debian-1207-bookworm-amd64-base.tar.gz
~~~

This creates `/boot/efi`  as RAID, which does not make any sense in my opinion, since UEFI cannot read from a RAID (yes, I know about the 0.9 meta-data hack, but I am not willing to use it since it is shady).

My expectation would be to create two separate EFI partitions. Or I am already able to archive that and did I miss something?

**Comment by asciiprod** [comment]
Yes, this is intended and will create a RAID for ESP using the metadata 1.0 to allow UEFI firmware to read the ESP. Only Ubuntu supports multiple ESP natively and will be set up as such.
If you like to have separate ESPs for any other distro, then feel free to remove the RAID after installation.
